spec/Section 2 -- Language.md

```diff
@@ -308,9 +308,8 @@ mutation {
 **Query shorthand**
 
 If a document contains only one operation and that operation is a query which
-defines no variables and contains no directives then that operation may be
-represented in a short-hand form which omits the {`query`} keyword and operation
-name.
+has neither variables nor directives then that operation may be represented
+in a short-hand form which omits the {`query`} keyword and operation name.
```

Comment:
Can we clarify it even more, like `doesn't define variables or has directives applied to it`?

Reply:
That's what I went with first but it became a bit wordy, hence my comment on "verbose". But yes, I'll give it a go :+1: